tensorflow/python/ops/ctc_ops.py

```diff
@@ -42,6 +46,28 @@
 from tensorflow.python.util import nest
 from tensorflow.python.util.tf_export import tf_export
 
+import os
```

Comment:
The linter will complain; standard python imports need to be above all others

Reply:
Sure. Removed this unused import.